cartography/intel/aws/ec2/instances.py

```diff
@@ -254,8 +286,8 @@ def cleanup_ec2_instances(neo4j_session: neo4j.Session, common_job_parameters: D
 
 @timeit
 def sync_ec2_instances(
-    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], current_aws_account_id: str,
-    update_tag: int, common_job_parameters: Dict,
+        neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str],
```

Comment:
Info: not sure if these indentations would fail when we submit it to Cartography

Reply:
Should not be a problem. We will find out when we open the PR :)